website/docs/r/codeartifact_domain.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "CodeArtifact"
+layout: "aws"
+page_title: "AWS: aws_codeartifact_domain"
+description: |-
+  Provides a CodeArtifact Domain resource.
+---
+
+# Resource: aws_codeartifact_domain
+
+Provides a CodeArtifact Domains Resource.
+
+## Example Usage
+
+```hcl
+resource "aws_kms_key" "example" {
+  description = "domain key"
+}
+
+resource "aws_codeartifact_domain" "example" {
+  domain         = "example.com"
```

Comment:
`example.com` is probably not the best sample name as this will be used as a DNS name prefix. How about just `example`?